pq-crypto/README.md

```diff
@@ -48,4 +48,21 @@ and use the BIKE1_L1.const.kat from the above Additional_Implementation.2019.03.
 1. Add fuzz testing in `tests/fuzz/s2n_KEM_NAME_fuzz_test.c`
 1. Add formal verification in `tests/saw/KEM_NAME/verify.saw`
 1. Create a new `s2n_cipher_suite` in `tls/s2n_cipher_suites.c`
-1. Create a new `s2n_cipher_preferences` in `tls/s2n_cipher_prefrences.c` that uses the new cipher suite
\ No newline at end of file
+1. Create a new `s2n_cipher_preferences` in `tls/s2n_cipher_prefrences.c` that uses the new cipher suite
+
+## How to use PQ cipher suites
+1. Checkout s2n `git clone https://github.com/awslabs/s2n.git`
+1. Following the docs/USAGE-GUIDE.md build s2n
+1. Use the sample server and client in the bin directory:
+```bash
+# Terminal 1
+export LD_LIBRARY_PATH=/path-to-s2n/s2n/test-deps/openssl-1.1.1/lib:/path-to-s2n/s2n/test-deps/openssl-1.1.1/lib:/path-to-s2n/s2n/lib:/path-to-s2n/s2n/bin
```

Comment:
Nit: I'd make the path it's own variable so that users only need to make a one line change
```
export PATH_TO_S2N=/path/to/s2n/
export LD_LIBRARY_PATH=${PATH_TO_S2N}/test-deps/openssl-1.1.1/lib:${PATH_TO_S2N}/test-deps/openssl-1.1.1/lib:${PATH_TO_S2N}/lib:${PATH_TO_S2N}/bin
```

Reply:
Good idea.